### PR TITLE
Kuma kong

### PR DESCRIPTION
**Pull request by devadvocado**
### Full changelog

* [Implement ...]
* [Fix ...]

### Issues resolved

Fix #XXX

### CODE REVIEW

File: kubernetes/kuma-demo-kong.yaml [hunk 1]
```diff
@@ -0,0 +1,608 @@
...
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ingress-kong
+  template:
+    metadata:
+      annotations:
+        prometheus.io/port: "9542"
+        prometheus.io/scrape: "true"
+        kuma.io/gateway: enabled
```
Comment (yskopets) [inline kubernetes/kuma-demo-kong.yaml]: Please explain this new configuration bit in the README.md

Somewhere close to `31. Kong Gateway Integration`

If these annotations are the only reason why we're including the complete K4K YAML into `kuma-demo` repository, then we should use `kubectl kustomize` instead

File: kubernetes/README.md [hunk 2]
```diff
@@ -726,4 +726,60 @@ This is what the query on `envoy_http_downstream_cx_tx_bytes_total` will return:
...
\ No newline at end of file
+Kuma ships with an internal GUI that will help you visualize the mesh and its policies in an intuitive format. It can be found on port `:5683` on the control-plane pod. We port-forwarded this port [earlier](#10-now-we-will-port-forward-the-kuma-control-plane-so-we-can-access-it-with-kumactl) so now we can access the GUI at [http://localhost:5683/](http://localhost:5683/).
+
+### 31. Kong Gateway Integration 
+
+The `Dataplane` can now operate in Gateway mode. This way you can integrate Kuma with existing API Gateways like [Kong](https://github.com/Kong/kong). When you use a Dataplane with a service, both inbound traffic to a service and outbound traffic from the service flows through the Dataplane. This API Gateway should be deployed as any other service within the mesh. Use the `kuma-demo-kong.yaml` file to deploy [Kong for Kubernetes](https://github.com/Kong/kubernetes-ingress-controller):
```
Comment (yskopets) [inline kubernetes/README.md]: This paragraph is too terse for me:
* `Dataplane` in Gateway mode
* existing API Gateways
* This API Gateway

Too many "gateways" without explaining how they are different and what is role of each of them.

File: kubernetes/kuma-demo-kong.yaml [hunk 3]
```diff
@@ -0,0 +1,608 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```
[the remark this answers is not in the document]
Comment (devadvocado) [reply kubernetes/kuma-demo-kong.yaml]: This is a modified version from the original K4K GitHub repo version, so we need it here.

File: kubernetes/kuma-demo-kong.yaml (hunk 1 above)
Comment (devadvocado) [reply kubernetes/kuma-demo-kong.yaml]: also editing the namespace for all the resources along with the annotations

Comment (yskopets) [reply kubernetes/kuma-demo-kong.yaml]: `kubectl kustomize` is the right solution for all of this

**Pull request merged**